pages/linux/chmod.md

```diff
@@ -0,0 +1,24 @@
+# chmod
```

Comment:
Wait a moment. I didn't notice this before, but there seems to be a copy of the page in the `chmod` directory. Is this intentional?

Reply:
haha yeah, I do mean the `common` directory lol

Ah, I see. If they are the same except for the last example, it makes sense to merge them and go with the common syntax to reduce maintenance complexity.

Reply:
> Wait a moment. I didn't notice this before, but there seems to be a copy of the page in the chmod directory. Is this intentional?

Yes, it is intentional. ;)

Reply:
But I'd assume that osx also accepts long options. So maybe we should move the `common` page with `-R` to `windows` and this one to `common`.

Reply:
It depends on what pages can be placed in `common/`. Pages for commands that are available just on some subset of platforms (Windows, Mac OS, Linux) or commands that are available everywhere?

Reply:
There are almost no commands, which are available on all platforms (e.g. Android). Typically if a command is available on at least two platforms it goes to `common`.

Reply:
Conversation is being closed as it's okay to duplicate pages to show different short and long options.

[Rules for long options](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax)
[When page duplicates can be created](https://github.com/tldr-pages/tldr/blob/main/CLIENT-SPECIFICATION.md#platform)